map/src/components/resultDetail.tsx

```diff
@@ -0,0 +1,247 @@
+import { ContactDetails } from '@reach4help/model/lib/markers';
+import React from 'react';
+import { MdEmail, MdLanguage, MdPhone } from 'react-icons/md';
+import { MarkerIdAndInfo } from 'src/components/map';
+import { Language, t } from 'src/i18n';
+
+import styled from '../styling';
+import MarkerType from './marker-type';
+
+interface Props {
+  result: MarkerIdAndInfo;
+  lang: Language;
+  fullDetail?: boolean;
+}
+
+const Div = styled('div')`
+  overflow-y: auto;
+  max-height: 100%;
+  background: #fff;
+  padding: ${p => p.theme.spacingPx}px;
+  overflow-y: auto;
+  background: #fff;
+  box-shadow: 0px 4px 12px rgba(0, 0, 0, 0.15);
```

Comment:
Do we have theme colours we can use for these instead of defining them?

Reply:
@jensren will check the theme, i believe this styling code was already there

Reply:
will leave this one out for next PR, it seems like the white color is there in the theme with the name `background`, but noticed #fff is being used at most of the places